src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -46,14 +47,30 @@ const ProfileInfoTable = ({
     'margin-bottom--1',
   ]);
   const tableRowTitleClassesMedium = prefixUtilityClasses(
-    ['margin-bottom--0', 'margin-right--2'],
```

Comment:
I'm a little concerned with getting rid of this. It'll be possible for the row's title on the left to run right into the row's value on the right. It doesn't happen right now with our row title's but it could: 
![image](https://user-images.githubusercontent.com/20728956/87952991-52ea1d80-ca5f-11ea-9014-3a04a6dd644e.png)

Reply:
Yup! I will add the right margin back, good catch!